tools/sigh.ts

```diff
@@ -62,6 +66,8 @@ const steps: {[index: string]: ((args?: string[]) => boolean)[]} = {
   peg: [peg, railroad],
   railroad: [railroad],
   test: [peg, railroad, build, runTests],
+  testShells: [peg, railroad, build, webpack, devServerAsync, testWdioShells],
+  testWdioShells: [testWdioShells],
```

Comment:
Is it actually worth having two commands? It looks to me like testWdioShells might not work properly unless you do all the extra steps that testShells does? Should we just stick with testShells only?

Reply:
I thought a lot about that. The existing setup lets you start a server and then run tests against it, without having to bounce it up and down. I thought I'd keep that, just in case. Scott is on vacation, or I'd ask him if he ever works that way.